### PR TITLE
[ DSD-1969 ] merged collab with qa-1201-b2 branch

### PR DESCRIPTION
**Pull request by syedsalman3753**

### CODE REVIEW

File: id-repository-default.properties [hunk 1]
```diff
@@ -26,7 +26,7 @@ mosip.idrepo.db.vid.db-name=mosip_idmap
 mosip.idrepo.db.vid.username=idmapuser
 mosip.idrepo.db.vid.password=${db.dbuser.password}
 mosip.idrepo.objectstore.account-name=idrepo
-mosip.idrepo.objectstore.bucket-name=idrepository
+mosip.idrepo.objectstore.bucket-name=idrepo
```
Comment (loganathan-sekaran) [inline id-repository-default.properties:29]: This whole file change is not latest. Please revert.

File: ida-partner.cer [hunk 2]
```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
+MIIDrDCCApSgAwIBAgIIkcTLqBQn22gwDQYJKoZIhvcNAQELBQAwdjELMAkGA1UE
```
Comment (loganathan-sekaran) [inline ida-partner.cer]: Why do we store a ida-partner.cer in config? I suggest to remove it, if needed take it from ida-internal service's getCertificate endpoint.

**Review by loganathan-sekaran — CHANGES_REQUESTED** [review]

File: admin-default.properties [hunk 3]
```diff
@@ -236,17 +237,17 @@ LOST_RID_API=${mosip.regproc.status.service.url}/registrationprocessor/v1/regist
...
+mosip.role.admin.getbulkuploadtranscationtranscationid=GLOBAL_ADMIN
+mosip.role.admin.getbulkuploadgetalltransactions=GLOBAL_ADMIN
 mosip.role.admin.postauditmanagerlog=GLOBAL_ADMIN,ZONAL_ADMIN
 
 
-mosip.role.admin.getgeneratecsrcertificateapplicationidreferenceid=KEY_MAKER
-mosip.role.admin.postuploadcertificate=KEY_MAKER
-mosip.role.admin.postgeneratecsr=KEY_MAKER
-mosip.role.admin.postuploadotherdomaincertificate=KEY_MAKER
+mosip.role.admin.getgeneratecsrcertificateapplicationidreferenceid=GLOBAL_ADMIN,ZONAL_ADMIN
```
Comment (ghost) [inline admin-default.properties]: seems like roles are not upto date in qa-1201-b2 branch. 
FYI @syedsalman3753  https://github.com/mosip/mosip-config/blob/develop-v3/admin-default.properties#L246
Existing collab values are valid. No need to override it here.

same comment applies to below properties:
mosip.role.admin.postuploadcertificate
mosip.role.admin.postgeneratecsr
mosip.role.admin.postuploadotherdomaincertificate

File: registration-default.properties [hunk 4]
```diff
@@ -403,31 +454,70 @@ mosip.kernel.uin.length.repeating-limit=${mosip.kernel.uin.length.repeating-limi
...
+mosip.registration.max_age=200
+
+# Parameters required for Backward Compatibility
+# These parameters needed for supporting previous versions of registration clients in case of upgraded env.
+
+mosip.registration.mdm.hostProtocol=http
+mosip.mdm.enabled=TRUE
+mosip.registration.ui_sync_data=y
+mosip.registration.Login_Credentials_Sync.frequency=190
+mosip.registration.consent_fra="Je comprends que les données recueillies à mon sujet lors de la enregistrement par ladite autorité comprennent mon -  <br> &#x2022 nom <br>&#x2022 Date de naissance <br>&#x2022 genre <br>&#x2022 adresse <br>&#x2022 coordonnées <br>&#x2022 Documents <br>Je comprends également que ces informations seront stockées et traitées dans le but de vérifier mon identité afin d'accéder à divers services, ou de se conformer à une obligation légale. Je donne mon consentement pour la collecte de ces données à cette fin."
```
Comment (ghost) [inline registration-default.properties]: Many unused properties are part of this file. This cleanup was done during 1.2.0 release.
Pls refer this https://github.com/mosip/mosip-config/blob/develop-v3/registration-default.properties

File: syncdata-default.properties [hunk 5]
```diff
@@ -343,6 +343,9 @@ datastores=ldap_1_DS,db_1_DS,db_2_DS
 ## Partner Management Service allowed partner domains
 mosip.kernel.partner.allowed.domains=AUTH,DEVICE,FTM
 
+## List of keys to auto generate.
+mosip.kernel.keymanager.autogen.appids.list=ROOT,KERNEL:SIGN,PRE_REGISTRATION,REGISTRATION,REGISTRATION_PROCESSOR,ID_REPO,KERNEL:IDENTITY_CACHE,RESIDENT,PMS
```
Comment (ghost) [inline syncdata-default.properties]: This is not required in syncdata-default.properties

File: kernel-default.properties [hunk 6]
```diff
@@ -412,11 +406,6 @@ syncdata.websub.resubscription.delay.millis=48000000
 ## We can provide multiple values with ";" separated 
 mosip.admin.ui.configs=version:${aplication.configuration.level.version};locationHierarchyLevel:${mosip.recommended.centers.locCode};mandatoryLanguages:${mosip.mandatory-languages};optionalLanguages:${mosip.optional-languages};supportedLanguages: ${mosip.mandatory-languages},${mosip.optional-languages};leftToRightOrientation:${mosip.left_to_right_orientation};rightToLeftOrientation:${mosip.right_to_left_orientation};countryCode:${mosip.country.code};filterValueMaxRecords:${mosip.kernel.filtervalue.max_columns};filterValueMaxCount:{"default":${mosip.kernel.filtervalue.max_columns},"registrationcenters":50,"locations":100}
 
-# Flag to identify the support of no thumbprint in 1.1.3 version. 
-# Added this for backward compatability. default is false, means support is not required.
-# Make it to true if support is required.
-mosip.kernel.keymanager.113nothumbprint.support=false
```
Comment (ghost) [inline kernel-default.properties:418]: why is this removed?

File: ida-partner.cer (hunk 2 above)
Comment (syedsalman3753) [reply ida-partner.cer]: removed the file

File: registration-default.properties (hunk 4 above)
Comment (syedsalman3753) [reply registration-default.properties]: updated as per develop-v3

File: syncdata-default.properties (hunk 5 above)
Comment (syedsalman3753) [reply syncdata-default.properties]: removed

File: kernel-default.properties (hunk 6 above)
Comment (syedsalman3753) [reply kernel-default.properties:418]: reverted this change

**Review by MonobikashDas — APPROVED** [review]
